src/thumbnailer_near_lossless.cc

```diff
@@ -14,6 +14,12 @@
 
 #include "thumbnailer.h"
 
+// List of pre-processing values used in binary search for near-lossless to
+// speed-up the algorithm. It is not necessary to binary search for all the
+// values in range [0,100] since actually for near-lossless, the frame size and
+// PSNR are not changed when the preprocessing increases a small quantity.
+const int preprocessing_list[6] = {0, 20, 40, 60, 80, 100};
```

Comment:
rename it kPreprocessingList to follow the styleguide
also put it in an anonymous namespace inside the libwebp namespace:
namespace libwebp {
namespace {
  const int preprocessing_list[6] = {0, 20, 40, 60, 80, 100};
}  // namespace

This is to avoid possible conflicts with constants in other files, even if it's fairly unlikely.